### PR TITLE
Run Code Cleanup

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateAggregate.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Data.Sqlite
     partial class SqliteConnection
     {
         /// <summary>
-        /// Creates or redefines an aggregate SQL function.
+        ///     Creates or redefines an aggregate SQL function.
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateAggregate.cs]: Change the .tt file

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateFunction.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Data.Sqlite
     partial class SqliteConnection
     {
         /// <summary>
-        /// Creates or redfines a SQL function.
+        ///     Creates or redfines a SQL function.
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateFunction.cs]: Change the .tt file

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateAggregate.cs (hunk 1 above)
Comment (smitpatel) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateAggregate.cs]: Updated.

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateFunction.cs (hunk 2 above)
Comment (smitpatel) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnection.CreateFunction.cs]: Updated.

**Pull request merged**